docs/scos/dev/front-end-development/202212.0/oryx/oryx-versioning.md

```diff
@@ -72,3 +72,30 @@ Minor releases are fully backward compatible and do not require any developer as
 ### Deprecation practices
 
 If a newer and better alternative to a feature is introduced in a minor release, the old one is deprecated. Deprecated features remain functional until the next major release, in which they are removed completely.
+
+## Feature Flag Versioning
+
+### Overview
+
+Oryx introduces Feature Flag Versioning to add new functionalities without affecting backward compatibility. This is achieved using the `ORYX_FEATURE_VERSION` environment variable.
```

Comment:
```suggestion
Oryx introduces Feature Flag Versioning to add new functionalities without affecting backward compatibility. This allows you to use new features before they are released in the next major version. This requires an explicit opt-in, using the `ORYX_FEATURE_VERSION` environment variable.
```

Reply:
I'd add a bit more intro here:
- what are typical things that we add with the feature version (like page structure, new component options, etc.)
- how does the versioning work in relation to major and minor (and not on patch)